applications/solvers/additiveFoam/thermo/TEqn.H

```diff
@@ -69,4 +69,13 @@
             break;
         }
     }
+
+    // set solid fraction consistent with temperature
```

Comment:
this is what fixed the issues during coarsening?

Reply:
But this also means that alpha1 is not forced to 1 when T < Tsol, or 0 when T > Tliq, so we must apply this explicit correction.